### PR TITLE
Make Ignitor test more strongly typed, and fix failure introduced with custom event args change

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Technically we could just fix the anonymously-typed data to match the new protocol, but this will be more resistant to similar problems in the future.

### CODE REVIEW

File: src/Components/Ignitor/src/ElementNode.cs [hunk 1]
```diff
@@ -100,11 +101,11 @@ public Task ClickAsync(HubConnection connection)
                 Type = clickEventDescriptor.EventName,
                 Detail = 1
             };
-            var webEventDescriptor = new
+            var webEventDescriptor = new WebEventDescriptor
             {
                 BrowserRendererId = 0,
                 EventHandlerId = clickEventDescriptor.EventId,
-                EventArgsType = "mouse",
+                EventName = "mouse",
```
Comment (javiercn) [inline src/Components/Ignitor/src/ElementNode.cs:108]: click?

**Review by javiercn — APPROVED** [review]
LGTM, I only have one comment because we changed it in the original PR

File: src/Components/Ignitor/src/ElementNode.cs (hunk 1 above)
Comment (SteveSandersonMS) [reply src/Components/Ignitor/src/ElementNode.cs]: Well spotted - fixed :)

The test was still passing though.

File: src/Components/Ignitor/src/ElementNode.cs [hunk 2]
```diff
@@ -73,12 +74,12 @@ internal Task SelectAsync(HubConnection connection, string value)
                 Value = value
             };
 
-            var webEventDescriptor = new
+            var webEventDescriptor = new WebEventDescriptor
```
Comment (pranavkm) [inline src/Components/Ignitor/src/ElementNode.cs:77]: This is nice. No more guess work to get the right shape!

File: src/Components/Ignitor/src/ElementNode.cs [hunk 3]
```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Text.Json;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components.RenderTree;
 using Microsoft.AspNetCore.SignalR.Client;
 
 #nullable enable
```
Comment (pranavkm) [inline src/Components/Ignitor/src/ElementNode.cs:11]: Could you remove these since the project is annotated in entirety?

**Review by pranavkm — APPROVED** [review]

File: src/Components/Ignitor/src/ElementNode.cs (hunk 3 above)
Comment (SteveSandersonMS) [reply src/Components/Ignitor/src/ElementNode.cs:11]: I'd prefer not to run the CI gauntlet again just for this - is it OK if this is left until next time we work on Ignitor? It's only a minor weirdness in the sources.

Comment (pranavkm) [reply src/Components/Ignitor/src/ElementNode.cs:11]: Sure.

**Pull request merged**